dtfs-central-api/src/mapping/fee-record-mapper.test.ts

```diff
@@ -132,7 +133,9 @@ describe('fee record mapper', () => {
 
     it('maps the fee record entity fees paid to the fee record reported fees amount', () => {
       // Arrange
-      const feeRecordEntity = FeeRecordEntityMockBuilder.forReport(aUtilisationReport()).withFeesPaidToUkefForThePeriod(314.59).build();
+      const feeRecordEntity = FeeRecordEntityMockBuilder.forReport(new UtilisationReportEntityMockBuilder().build())
+        .withFeesPaidToUkefForThePeriod(314.59)
```

Comment:
can this number be a const, and the same for rest in this file.  const feesPaid = 314.59.  that way it only has to be changed once if we change the value for the test.

Reply:
That is out of scope of this ticket, I'm just replacing `aUtilisationReport()`